tfx/examples/iris/iris_utils.py

```diff
@@ -206,6 +208,16 @@ def run_fn(fn_args: executor.TrainerFnArgs):
   """
   schema = io_utils.parse_pbtxt_file(fn_args.schema_file, schema_pb2.Schema())
 
+  # TODO(b/160795287): Deprecate estimator based executor.
+  # Modify fn_args with model_run_dir given as the working directory.
+  # Executor will then copy user models to model artifact directory.
+  serving_dest = fn_args.serving_model_dir
+  eval_dest = fn_args.eval_model_dir
```

Comment:
change to _trainer_fn, and do it like testdata/module file (avoid updating fn_args in module file)

Reply:
unsolved?